apps/sdk-vite-integration/tests/Hash.spec.tsx

```
@@ -1,9 +1,9 @@
 import { expect, test } from 'vitest'
 import { render } from 'vitest-browser-react'
-import Hash from '../src/components/Hash'
+import Hash from '../src/components/Hash.js'
```

ESLint

[error] Insert `;`

(prettier/prettier)